crepecake/src/main/java/com/google/cloud/tools/crepecake/image/json/JsonToImageTranslator.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.crepecake.image.json;
+
+import com.google.cloud.tools.crepecake.blob.BlobDescriptor;
+import com.google.cloud.tools.crepecake.image.DescriptorDigest;
+import com.google.cloud.tools.crepecake.image.DigestOnlyLayer;
+import com.google.cloud.tools.crepecake.image.DuplicateLayerException;
+import com.google.cloud.tools.crepecake.image.Image;
+import com.google.cloud.tools.crepecake.image.Layer;
+import com.google.cloud.tools.crepecake.image.LayerCountMismatchException;
+import com.google.cloud.tools.crepecake.image.LayerPropertyNotFoundException;
+import com.google.cloud.tools.crepecake.image.ReferenceLayer;
+import com.google.cloud.tools.crepecake.image.ReferenceNoDiffIdLayer;
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO: Add translation of container configuration config.
+/** Translates {@link V21ManifestTemplate} and {@link V22ManifestTemplate} into {@link Image}. */
+public abstract class JsonToImageTranslator {
```

Comment:
private constructor?

Reply:
just need this addressed and then we're good to go?

Reply:
This has already been addressed via https://github.com/GoogleCloudPlatform/minikube-build-tools-for-java/pull/72/commits/b29fdc5214b5910ec96c30373c7f0c5829b2cd97

Reply:
ah okay :D